pyproject.toml

```diff
@@ -44,13 +44,40 @@ dev = [
     "tox",
 ]
 tests = [
+    "pytest>=6.0",
+    "pytest-split",
+    "pytest-dotenv",
+    "requests-mock",
+    "pytest-cov",
+    "pytest-describe",
     "pre-commit"
 ]
 
+######################################
+# TESTING
+######################################
+
 [tool.hatch.envs.tests]
 dependencies = [
-    "dag-factory[tests]"
-    ]
+    "dag-factory[tests]",
+    "apache-airflow~={matrix:airflow}.0,!=2.9.0,!=2.9.1",  # https://github.com/apache/airflow/pull/39670
```

Comment:
this should be just >=2.2?

Reply:
test matrix values are accepted and used here for corresponding Airflow version environments using hatch.